src/dex-ui/services/HederaService/GovernorService.ts

```diff
@@ -147,15 +147,15 @@ const sendCreateTransferTokenProposalTransaction = async (
   const transferToAddress = AccountId.fromString(accountToTransferTo).toSolidityAddress();
   const tokenToTransferAddress = TokenId.fromString(tokenToTransfer).toSolidityAddress();
   const contractCallParams = new ContractFunctionParameters()
-    /** This is 'description' on the contract function */
     .addString(title)
+    .addString(description)
+    .addString(linkToDiscussion)
     .addAddress(transferFromAddress)
     .addAddress(transferToAddress)
     .addAddress(tokenToTransferAddress)
-    .addInt256(amountToTransfer)
-    /** This is 'title' on the contract function */
-    .addString(description)
-    .addString(linkToDiscussion);
+    .addInt256(amountToTransfer);
+  return Promise.reject();
```

Comment:
Looks like early return code below will not work or this is done for testing purpose.?

Reply:
Sorry, my bad!